### PR TITLE
Extend parameter description pattern to stop for single line comment

### PR DESCRIPTION
**Pull request by DZunke**
When utilizing single line comments the description of the argument in the toolbox contains `*/` of the closing comment. I have changed the regex to also have this supported. Also added new tests for the `ParameterAnalyzer` to check the parsing works as expected for different type of doc blocs.

### CODE REVIEW

File: tests/ToolBox/ParameterAnalyzerTest.php [hunk 1]
```diff
@@ -163,4 +164,76 @@ public function detectParameterDefinitionNone(): void
...
+            'expectedResult' => '',
+        ];
+
+        yield 'single line doc block with description' => [
+            'docComment' => '/** @param string $myParam The description */',
+            'expectedResult' => 'The description',
+        ];
+
+        yield 'multi line doc block with description and other tags' => [
+            'docComment' => '/** @param string $myParam The description
```
Comment (OskarStark) [inline tests/ToolBox/ParameterAnalyzerTest.php:218]: What about using php multilingual variable for readability here?

**Review by OskarStark — COMMENTED** [review]
Thanks 🙏

File: tests/ToolBox/ParameterAnalyzerTest.php (hunk 1 above)
Comment (chr-hertel) [reply tests/ToolBox/ParameterAnalyzerTest.php:218]: Preeeetty sure he meant multiline not multilingual :D

Comment (OskarStark) [reply tests/ToolBox/ParameterAnalyzerTest.php:218]: Yes, sorry 🙈 autocorrection...

Comment (DZunke) [reply tests/ToolBox/ParameterAnalyzerTest.php]: Hehe. Yeah. Always hard to teach the smartphone. I have changed the texts to a nowdoc format. This should improve the readability.

**Review by chr-hertel — APPROVED** [review]
Thank you @DZunke!

**Pull request merged**